index.html

```diff
@@ -633,10 +633,27 @@ <h2>Core Data Model</h2>
 
       <p>
 The following sections outline core data model concepts, such as <a>claims</a>,
-<a>credentials</a>, and <a>presentations</a>, which form the foundation of this
+<a>credentials</a>, <a>presentations</a>, <a>verifiable credentials</a>, and
+<a>verifiable presentations</a>, which form the foundation of this
 specification.
       </p>
 
+      <p class="note"
+        title="Abstract concepts vs. concrete concepts">
+Readers might note that some abstract concepts described in this section
+do not have normative requirements nor associated media types defined by this
```

Comment:
The normative requirements for a verifiableCredential also apply to the credential ie. the VC without the proof. So I do not agree with the sentence above. The difference is that the credential does not have a standardised media type whilst the verifiableCredential does have. (As @TallTed  points out, someone could specify a media type for a credential as well if they wanted to).

Reply:
@dlongley just asserted above that the media type for a "credential" could be `application/vc+ld+json`... so I don't know if we have consensus yet.

Reply:
> The media type for a credential could be `application/json`

I would prefer `application/ld+json`, as `application/json` is likely to encourage credentials which require securing as arbitrary JSON, as that is what they are...

Reply:
> @dlongley just asserted above that the media type for a "credential" could be `application/vc+ld+json`... so I don't know if we have consensus yet.

No, I don't think he did... I think he was saying (in the very least): "Don't make it impossible to add a proof to a chain of proofs via a issuer service." ... in that situation, you would send an `application/vc+ld+json` ... but I would expect a number of issuer implementations to accept `application/json`, `application/ld+json`, or `application/vc+ld+json` as long as it could process it correctly. Developers often tend to be lazy/wrong about media types... which is why it's difficult to build any software system that depends on them.